subprojects/packagefiles/libgpiod/bindings/python/meson.build

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# Copyright 2021 IBM Corp.
+
+python3_dep = dependency('python3', required: true)
+python3_prog = find_program('python3')
+python3_config_prog = find_program('python3-config')
+if not python3_config_prog.found()
+    error('python3-config not found - needed for python bindings')
+endif
+
+# TODO: Set up PYTHON_CPPFLAGS and PYTHON_LIBS
+gpiodpy = shared_module('gpiod', 'gpiodmodule.c',
```

Comment:
I think you probably want https://mesonbuild.com/Python-module.html#extension_module

Reply:
Ah neat. Ack.